cpg_workflows/jobs/mito.py

```diff
@@ -0,0 +1,328 @@
+            coverage at this value.
+
+    Outputs:
+        metrics: output file
+        theoretical_sensitivity: Undocumented CollectWgsMetrics output?
+    """
+    job_attrs = job_attrs or {}
+    j = b.new_job('collect_coverage_metrics', job_attrs)
+    j.image(image_path('picard'))
+
+    res = STANDARD.request_resources(ncpu=2)
+    res.set_to_job(j)
+
+    cmd = f"""
+        picard \
+            CollectWgsMetrics \
+            INPUT={cram.cram} \
+            VALIDATION_STRINGENCY=SILENT \
+            REFERENCE_SEQUENCE={reference.base} \
+            OUTPUT={j.metrics} \
+            USE_FAST_ALGORITHM=true \
+            READ_LENGTH={read_length_for_optimization} \
+            COVERAGE_CAP={coverage_cap} \
+            INCLUDE_BQ_HISTOGRAM=true \
+            THEORETICAL_SENSITIVITY_OUTPUT={j.theoretical_sensitivity}
+
+    """
+
+    j.command(command(cmd))
+    if metrics:
+        b.write_output(j.metrics, str(metrics))
+    if metrics:
+        b.write_output(j.theoretical_sensitivity, str(theoretical_sensitivity))
+
+    return j
+
+
+def extract_coverage_mean(
+    b,
+    metrics: hb.ResourceFile,
+    mean_path: Path | None = None,
+    median_path: Path | None = None,
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    Extract mean and median coverage values for sequencing group
+
+    Args:
+        metrics: CollectWgsMetrics metrics output to process.
+        mean_path: Output path for mean file.
+        median_path: Output path for median file.
+
+    Outputs:
+        job.mean_coverage: mean coverage of chrM
+        job.median_coverage: median coverage of chrM
+    """
+    job_attrs = job_attrs or {}
+    j = b.new_job('extract_coverage_mean', job_attrs)
+    j.image(image_path('peer'))
+
+    res = STANDARD.request_resources(ncpu=2)
+    res.set_to_job(j)
+
+    cmd = f"""
+
+    R --vanilla <<CODE
+        df = read.table(
+            "{metrics}",skip=6,header=TRUE,stringsAsFactors=FALSE,sep='\\\\t',nrows=1
+        )
+        write.table(
+            floor(df[,"MEAN_COVERAGE"]),
+            "{j.mean_coverage}",
+            quote=F, col.names=F, row.names=F)
+        write.table(
+            df[,"MEDIAN_COVERAGE"],
+            "{j.median_coverage}",
+            quote=F, col.names=F, row.names=F)
+        CODE
+    """
+
+    j.command(command(cmd))
+    if mean_path:
+        b.write_output(j.mean_coverage, str(mean_path))
+    if median_path:
+        b.write_output(j.median_coverage, str(median_path))
+
+    return j
+
+
+def coverage_at_every_base(
+    b,
+    cram: hb.ResourceGroup,
+    reference: hb.ResourceGroup,
+    intervals_list: hb.ResourceFile,
+    job_attrs: dict | None = None,
+) -> Job:
+    """
+    Run picard CollectHsMetrics to calculate read coverage at each base.
+
+    Args:
+        cram: Input cram
+        reference: Cram reference genome
+        intervals_list: intervals list of target region
+
+    Outputs:
+        job.per_base_coverage
+        job.hs_metics_out
```

Comment:
I can see this is an unused output but just a small typo fix: `job.hs_metrics_out`